app/templates/public-services/add.hbs

```diff
@@ -152,14 +152,12 @@
         {{else if this.hasResults}}
           <Content.body as |concept|>
             <td>
-              <AuLink
-                @route="public-services.concept-details"
```

Comment:
It seems it's now no longer possible to preview a concept before creating an instance. Was that an intentional change?

_if_ skip the preview, I think that only makes sense for concepts that are already instantiated? So we could conditionally show the preview link if there are no instances yet, or the button that triggers the modal if there are already some instances.

Reply:
It was indeed an oversight, I wasn't sure what it did. I've added the preview back in

Reply:
I would do it conditionally here, instead of in the modal. That way the API call in the modal doesn't need to happen when it isn't needed and links are generally better because users can open those in new tabs if they want.

```hbs
{{#if concept.displayConfiguration.isInstantiated}}
  {{! modal button }}
 <AuButton ...
{{else}}
  {{! the original AuLink component}}
  <AuLink ...
{{/if}}
```

But I'm still not convinced we should display the modal here (or a modal at all). I think it might be better to update the details preview page instead, but again, ideally this would get looked at by the design team..

Reply:
I'm not sure why but the isInstantiated doesn't seem to be accurate, some of the instances are displayed correctly but some are not..

Reply:
Hmm, that's a bigger problem then. I can't remember how that part works exactly, or where it's updated. But that would also mean that users would see the modal even if there is no "toegevoegd" label next to the concept. :thinking:

Reply:
Yes indeed,  I have not really used the app before so I'm not sure if I'm missing something (maybe it needs some time to update?)

Reply:
Cc: @mirdono